packages/react/src/example/JupyterExample.tsx

```diff
@@ -138,23 +138,25 @@ const root = createRoot(div)
 
 root.render(
   <Jupyter lite={false} terminals={true}>
-    <Console/>
-    <hr/>
-    <CellPreview/>
-    <CellToolbar/>
-    <Cell/>
-    <hr/>
-    <Outputs/>
-    <hr/>
-    <NotebookToolbar/>
-    <Notebook
-      path="test.ipynb"
-//      model={notebookExample as INotebookContent}
-      CellSidebar={CellSidebarDefault}
+    <Console />
+    <hr />
+    <CellPreview />
+    <CellToolbar />
+    <Cell />
+    <hr />
+    <Outputs />
+    <hr />
+    <NotebookToolbar />
+    <div style={{ height: "100vh", width: "100vw" }}>
```

Comment:
Would it be possible to move that `div` to the Notebook.tsx component so every consumer get that for free and does not have to write it down each time?